firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallationException.java

```diff
@@ -12,43 +12,45 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package com.google.firebase.installations.remote;
+package com.google.firebase.installations;
 
 import androidx.annotation.NonNull;
 import com.google.firebase.FirebaseException;
 
-/** The class for all Exceptions thrown by {@link FirebaseInstallationServiceClient}. */
-public class FirebaseInstallationServiceException extends FirebaseException {
+/** The class for all Exceptions thrown by {@link FirebaseInstallations}. */
+public class FirebaseInstallationException extends FirebaseException {
 
   public enum Code {
```

Comment:
As this exception is exposed to the caller(apps), I am not sure if we need to expose those detailed code to them, or just expose something they may care about, for others, we can just uniform as "SDK_INTERNAL_ERROR".

Reply:
Done. Thanks.